chainer/functions/parameter.py

```diff
@@ -22,6 +22,13 @@ def __init__(self, array):
         self.W = array
         self.gW = numpy.empty_like(array)
 
+    def __call__(self, volatile=False):
+        ret = super(Parameter, self).__call__()
```

Comment:
When `volatile`  is `False`, I think `rank` is set to `0`. Is this behavior acceptable?